multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -18,6 +20,28 @@
 automm_presets = Registry("automm_presets")
 matcher_presets = Registry("matcher_presets")
 
+default_hyperparameter_tune_kwargs = {
+    "searcher": "bayes",
+    "scheduler": "ASHA",
+    "num_trials": 512,
+}
+
+default_tunable_hyperparameters = {
+    "optimization.learning_rate": tune.loguniform(1e-5, 1e-2),
+    "optimization.optim_type": tune.choice(["adamw", "sgd"]),
+    "optimization.max_epochs": tune.choice(list(range(5, 31))),
+    "env.batch_size": tune.choice([32, 64, 128, 256]),
+}
+
+
+def parse_presets_str(presets: str):
+    use_hpo = False
+    if presets.endswith("_hpo"):
```

Comment:
will this be case sensitive?

Reply:
`presets` is already converted to lower case in predictor init.